IOSExercises/Classes/EXRColor.h

```diff
@@ -7,14 +7,23 @@
 //
 
 #import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NS_ENUM(NSInteger, EXRSupportedColor) {
+  EXRSupportedColorRed,
```

Comment:
```suggestion
  EXRSupportedColorRed = 0,
```

Initialize the first enum type (no need to do so for succeeding enum types)